### PR TITLE
NEW-UI-Edit profile page

### PR DESCRIPTION
**Pull request by yash15601**
### Screenshots of the changes (If any) -
DESKTOP:
Screenshot (75)

MOBILE:
image

### CODE REVIEW

**Comment by yash15601** [comment]
@nitin10s  @aayush-05  Review

**Comment by yash15601** [comment]
@nitin10s @aayush-05  @tachyons  Review please

**Comment by tachyons** [comment]
Do not override bootstrap classes. Wrap them inside a div if you really want to

**Comment by yash15601** [comment]
@tachyons  their is some need to override the bootstrap class due to padding and margin if we wrap it into a new div we can,t acheive the look we wanted so using multiple div is not a soultion i tryed before it will not work on mobile view so i used some media queries  to achive the mobile view

**Comment by tachyons** [comment]
Problem is that styles we add here will reflect on entire app, not just profile page

**Comment by yash15601** [comment]
@tachyons i will try not to overide working on it

**Comment by aayush-05** [comment]
Try doing this, say in the element where you have btn-secondary; add another class btn-secondary-profile_page. This way you can use the bootstrap classes, as well as add any custom styles you want to without changing the default class.
One more query, when you are already using container and row classes from bootstrap. They why are you using manual media queries rather than the col class ?

**Comment by yash15601** [comment]
@aayush-05 almost i fixed the thing that are overriding bootstrap other in progress

**Comment by yash15601** [comment]
@aayush-05  review  done without overiding  bootstrap classes ,any changes by your side?

**Comment by yash15601** [comment]
@tachyons  review please ,no overiding bootstrap classes , no use of media quires as @aayush-05 said used rows and columns ,i think everything is working fine

**Comment by yash15601** [comment]
@nitin10s Review please

**Comment by yash15601** [comment]
@tachyons ??

File: app/assets/stylesheets/layout.scss [hunk 1]
```diff
@@ -1,4 +1,56 @@
 // All the custom class styles
+.noborder {
```
Comment (tachyons) [inline app/assets/stylesheets/layout.scss]: `border-0` is available from bootstrap

File: app/assets/stylesheets/layout.scss [hunk 2]
```diff
@@ -1,4 +1,56 @@
...
+  border: 0;
+  border-radius: 0;
+  padding: 1.25rem;
+}
+
+.astrikstyle {
```
Comment (tachyons) [inline app/assets/stylesheets/layout.scss]: `astrik-style`
Also is this common to entire CV ?

File: app/assets/stylesheets/layout.scss [hunk 3]
```diff
@@ -1,4 +1,56 @@
...
+  color: $danger;
+  font-size: 10px;
+}
+
+.aligncenter {
```
Comment (tachyons) [inline app/assets/stylesheets/layout.scss]: There is already a built in bootstrap class to align center, what does this do ?

File: app/assets/stylesheets/layout.scss [hunk 4]
```diff
@@ -1,4 +1,56 @@
...
+  color: $pure-white;
+  padding: .5em 3em;
+}
+
+.profile {
+  background-color: $slightblack;
+  border: 0;
+}
+
+.required {
```
Comment (tachyons) [inline app/assets/stylesheets/layout.scss]: core bootstrap class, Whats is the purpose of overriding here ?

File: app/assets/stylesheets/layout.scss [hunk 5]
```diff
@@ -1,4 +1,56 @@
...
+  color: $danger;
+  font-size: 30px;
+}
+
+.boldtext {
```
Comment (tachyons) [inline app/assets/stylesheets/layout.scss]: bootstrap already have class for this

File: app/assets/stylesheets/layout.scss [hunk 6]
```diff
@@ -1,4 +1,56 @@
...
+  border: 1px solid $green;
+  border-radius: 6px;
+  cursor: pointer;
+  display: inline-block;
+  padding: 6px 12px;
+  text-align: center;
+  width: 155px;
+}
+
+.filebtn {
```
Comment (tachyons) [inline app/assets/stylesheets/layout.scss]: `.file-button`

**Comment by yash15601** [comment]
@tachyons  done as you said i made some changes review  please

File: app/assets/stylesheets/layout.scss [hunk 7]
```diff
@@ -1,4 +1,34 @@
 // All the custom class styles
+.edit-profile-page-astrikstyle {
+  color: $danger;
+  font-size: 10px;
+}
+
+.btn-secondary-profile-page {
```
Comment (tachyons) [inline app/assets/stylesheets/layout.scss]: should be in profile.scss

File: app/assets/stylesheets/profile.scss [hunk 8]
```diff
@@ -0,0 +1,14 @@
+.edit-profile-page-astrikstyle {
```
Comment (tachyons) [inline app/assets/stylesheets/profile.scss]: import this file from application.scss

Comment (yash15601) [reply app/assets/stylesheets/profile.scss]: @tachyons  can you elaborate i can't understand what to import please

Comment (yash15601) [reply app/assets/stylesheets/profile.scss]: Screenshot (76)
 like this?? @tachyons

**Comment by yash15601** [comment]
@tachyons  done.

**Comment by tachyons** [comment]
Thanks for the patience :-)

**Comment by yash15601** [comment]
@tachyons  thank you finally 😊

**Comment by aayush-05** [comment]
Hey!
Can you fix this (button on hover) and maybe decrease the font size as well (I might be nitpicking at this).
image
_(You can use the font on this page as reference)_
image

**Comment by yash15601** [comment]
@aayush-05  will do and fix this

**Comment by nitin10s** [comment]
@aayush-05 We'll have to set some standard practices on how the buttons should look on hover, etc. We can leave this, for now, I'll say.

**Comment by yash15601** [comment]
@nitin10s you are right but i have solved  the above bug as @aayush-05 said can i create new pr their are  some  minor changes  or leave for now

**Comment by aayush-05** [comment]
> @aayush-05 We'll have to set some standard practices on how the buttons should look on hover, etc. We can leave this, for now, I'll say.

Sure, we will have to setup a design guidelines manual to maintain uniformity.

**Comment by yash15601** [comment]
@aayush-05 agree with both of you should have some standard to design

**Pull request merged**